integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/preaaggregate/PreAggregateTableHelper.scala

```diff
@@ -110,7 +110,29 @@ case class PreAggregateTableHelper(
     // Datamap table name and columns are automatically added prefix with parent table name
     // in carbon. For convenient, users can type column names same as the ones in select statement
     // when config dmproperties, and here we update column names with prefix.
-    val longStringColumn = tableProperties.get(CarbonCommonConstants.LONG_STRING_COLUMNS)
+    // If longStringColumn is not present in dm properties then we take long_string_columns from
+    // the parent table.
+    var longStringColumn = tableProperties.get(CarbonCommonConstants.LONG_STRING_COLUMNS)
```

Comment:
So user do not have to config dmproperties for long string anymore and this line is no use after this PR?

Reply:
The codes is not consistent with comments in line 113. Please fix it

Reply:
Fixed it. If the user doesn't configure long_string in dmproperties, we take long_string_cols from the parent table.

Reply:
fine